src/main/java/cn/nukkit/block/BlockConcretePowder.java

```diff
@@ -101,4 +102,8 @@ public boolean place(@Nonnull Item item, @Nonnull Block b, @Nonnull Block target
 
         return true;
     }
+
+    public DyeColor getDyeColor() {
+        return DyeColor.getByWoolData(getDamage());
```

Comment:
Use this as reference https://github.com/PowerNukkit/PowerNukkit/blob/bleeding/src/main/java/cn/nukkit/block/BlockGlassStained.java#L59